### DISCUSSION TITLE
Magventure Cool-40 (rat coil) file?

### DISCUSSION THREAD

**Discussion by NinetLab-Redux** [simnibs/simnibs#435] [8. Simulation workflows] [answered]
Hi,
I would like to model the e-field of the Cool-40 coil on mice, and I note in the paper (Alekseichuk, I., Mantell, K., Shirinpour, S. & Opitz, A. Comparative modelling of transcranial magnetic and electric stimulation in mouse, monkey, and human. Neuroimage 194, 136–148 (2019)) that SIMNiBS was used to do simulations on an equivalent coil. However, I could not find this coil on the list of coil files in SIMNiBS 4.0. 
Is there a way to access that coil file to test it, please?
Thanks
Fidel

**Comment by TorgeW** [comment, marked as answer]
The coil is not part of the coil selection of SimNIBS. 
It will be possible to create coils more easily with information of the exact geometry of the windings of the coil soon with the next SimNIBS version. 
Otherwise you will have to get in contact with the authors and ask if they have a model already.

**Reply by NinetLab-Redux** [reply]
Thanks for the quick reply; it is great to hear about this future feature. I'll reach out to the authors re Cool-40 coil.
